client/components/post-card/search.jsx

```diff
@@ -39,8 +39,10 @@ function SearchByline( { post, site, feed } ) {
 
 export function SearchPostCard( { post, site, feed, onClick = noop, onCommentClick = noop } ) {
 	const featuredImage = post.canonical_image;
+	const hasPost = !! post;
 	const classes = classnames( 'post-card__search', {
-		'has-thumbnail': !! featuredImage
+		'has-thumbnail': !! featuredImage,
+		'is-photo': hasPost && post.excerpt.length < 1
```

Comment:
we have a DisplayType for PhotoOnly. should be able to say `&& ( post.display_type & DisplayTypes.PHOTO_ONLY )`